docs/core/diagnostics/distributed-tracing-concepts.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Distributed tracing concepts - .NET
+description: .NET distributed tracing concepts
+ms.date: 03/14/2021
+---
+
+# Distributed Tracing
+
+Distributed tracing is a diagnostic technique that helps engineers localize failures and 
+performance issues within applications, especially those that may be distributed across
+multiple machines or processes. See the [Distributed Tracing Overview](distributed-tracing.md)
+for general information about where distributed tracing is useful and example code to get
+started.
+
+## Concepts
+
+### Traces and Activities
+
+Each time a new request is received by an application a new trace can be initiated. In
```

Comment:
the application can join the existing trace (if there is one from the incoming request) or a new trace can be initiated

Reply:
I reworded to this to say it can be associated with a trace without explicitly saying the trace is new. I worry that the reader doesn't have enough context yet to understand what it means to join an existing trace or that trace information flows with the request.